tests/spread/plugins/go/cross_compile_cgo/task.yaml

```diff
@@ -1,15 +1,18 @@
 summary: Cross-compile Go snaps using cgo
 
-# cgo doesn't build on 14.04
```

Comment:
Is this comment no longer valid? Did something change?

Reply:
trusty does not have a base, we are not testing it, so the comment is just as good as specifying where else it does not work